cli-scripts/attach-wallet

```diff
@@ -9,15 +9,15 @@
 #   - Example; myorg_issuer
 #
 # storageType:
-#   - The wallet storage type.
+#   - Optional - The wallet storage type.  Defaults to 'default'.
 #   - Remote Wallet Example; postgres_storage
 #   - Local Wallet Example; default
 #
 # storageConfig:
-#   - The wallet storage configuration.
+#   - Optional - The wallet storage configuration.  Defaults to '{}'.
 #   - Remote Wallet Example; '{"url":"192.168.65.3:5435"}'
 #   - Local Wallet Example; '{}'
 # ===================================================================================
 
-wallet attach ${walletName} storage_type=${storageType} storage_config=${storageConfig}
+-wallet attach ${walletName} storage_type=${storageType} storage_config=${storageConfig}
```

Comment:
Could you explain more about that little '-' here, because I keep getting error when trying to ru the export_wallet in my machine until I remove this '-'

Reply:
The `-` in `-wallet attach ...` stops the command from throwing an error and aborting the script if the wallet is already attached.

Reply:
Yeah, I pretty sure with the correct key, currently I could manage to do it run each step of the indy command (wallet create / wallet attach/ wallet open), but not by  ./manage

Reply:
Could you enter this as an issue, so we can track and possibly fix the issue.  Thanks

Reply:
Actually, what I tried to do a bit different with the guide, so I decided to submit an issue here https://github.com/bcgov/von-network/issues/119